accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -1227,6 +1236,59 @@ of an expression statement. This is similar to existing restrictions on map
 literals appearing in expression statements. In the rare case where a user
 really wants one there, they can parenthesize it.
 
+#### Function expression in guard ambiguity
+
+Function expressions also use `=>`, which leads to a potential ambiguity:
+
+```dart
+switch (obj) {
```

Comment:
This example looks like it introduces a switch statement rather than a switch expression, due to the "Expression statement ambiguity" resolution above.  How about if we make the example:

```dart
var x = switch(object) {
  ...
};
```
just so that it's clear that we're talking about a switch expression?

Reply:
Good catch, done.